powerauth-restful-security-spring-annotation/src/main/java/io/getlime/security/powerauth/rest/api/spring/provider/PowerAuthEncryptionProviderBase.java

```diff
@@ -136,6 +136,7 @@ public void decryptRequest(@Nonnull HttpServletRequest request, @Nonnull Type re
 
             // Prepare and validate EncryptedRequest object
             final EncryptedRequest encryptedRequest = new EncryptedRequest(
+                    null, // TODO Lubos - merge with Petr's branch
```

Comment:
Will be fix after merge of https://github.com/wultra/powerauth-restful-integration/pull/546